client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/utils/style-variations.ts

```diff
@@ -14,3 +14,26 @@ export function removeLegacyDesignVariations( designs: Design[] ) {
 		);
 	return designs.filter( ( design ) => ! isVariationOfAnotherDesign( design ) );
 }
+
+/**
+ * Promote designs with style variations by having them at the beginning of the list.
+ */
+export function promoteDesignVariations( designs: Design[] ) {
+	return designs.sort( ( a, b ) => {
+		const a_style_variations = a.style_variations || [];
```

Comment:
This code now looks correct :)

I have a suggestion to even simplify the logic, feel free to use or not, something like this

```
const a_has_variations = (a.style_variations || []).length > 0;
const b_has_variations = (b.style_variations || []).length > 0;

if (a_has_variations == b_has_variations) {
  return 0;
}
return a_has_variations ? 1 : -1;
```

Reply:
yes, this is what I thought 😆

Reply:
Ah right, sorry, untested code 😄